src/webpack.config.ts

```diff
@@ -116,6 +122,9 @@ function webpackConfig(args: Partial<BuildArgs>) {
 					result.request = result.request.replace(/\.m\.css$/, '.m.css.js');
 				}
 			}),
+			new DefinePlugin({
+				'process.env.DOJO_SERVICE_WORKERS': JSON.stringify(includeServiceWorker)
```

Comment:
Do we have a standard for env options? Should we put them all under one dojo namespace?